docs/src/content/docs/de/components/asides.mdx

```diff
@@ -132,6 +132,36 @@ Ein Warnhinweis *mit* einem benutzerdefinierten Titel.
 
 </Preview>
 
+### Benutzerdefinierte Symbole verwenden
+
+Überschreibe die Standard-Nebenfeldsymbole, indem du das Attribut [`icon`](#icon) auf den Namen [eines der in Starlight integrierten Symbole](/de/reference/icons/#alle-symbole) setzt.
```

Comment:
Is it intentional that this uses “Neben<em>feld</em>symbole” when the rest of the page uses “Neben<em>bemerkung</em>”? (Or “Hinweis” in a few places.)